xchange-hitbtc/src/test/java/org/knowm/xchange/hitbtc/service/trade/HitbtcTradeServiceRawTest.java

```diff
@@ -32,4 +47,20 @@ public void testReadSymbolDoge() throws Exception {
     assertEquals(Order.OrderType.ASK, HitbtcAdapters.readOrderType(id));
   }
 
+  @Ignore //Ignored because this executes a real trade.
+  @Test
+  public void testCreateAndCancelOrder() throws Exception {
```

Comment:
Creating tests for these requires testing in 'production'. Accidentally lost .01 BTC ($44) to a low trade... :-(